BM/runtests

```diff
@@ -422,8 +428,8 @@ while getopts ":o:d:f:c:h" opt; do
       ;;
     f)
       CMDFILES=$OPTARG
-      [[ "$CMDFILES" == "$TESTS_CLIENT" ]] && test_all "$TESTS_CLIENT"
-      [[ "$CMDFILES" == "$TESTS_SERVER" ]] && test_all "$TESTS_SERVER"
+      # If tests-server type will list tests-server in all subfolders
+      [[ "$CMDFILES" == *"-"* ]] && list_all_test_files "$CMDFILES"
```

Comment:
Hi @xupengfe, can we have another parameter to handle such batch testing, like '-b' ?
In that case, '-f' is for specified tests file, while runtests can run multiple tests with option '-b'.

Reply:
Hi @ysun ,

-f already support this function.
For example:  ./runtests -f cet/tests,umip/tests,cstate/tests-server
Is that you want?

It's already supported.

Thanks!

Reply:
I understand the usage of '-f <tests file>'. However, my point is that we may NOT reuse '-f' for multiple tests. Instead, create a new option for running multiple tests.

Reply:
Ok will use -t  tests-server and so on tests-xxx.   And -f for subfeature/tests.

Thanks!

Reply:
Done for it, and update the runtests  usage print also.

Thanks!